docs/apache-airflow/logging-monitoring/check-health.rst

```diff
@@ -72,6 +72,18 @@ status of the application. The return code is only indicative of the state of th
   monitoring, then in case of failure of the web server, you will lose the ability to monitor scheduler, which means
   that it can be restarted even if it is in good condition. For greater confidence, consider using :ref:`CLI Check for Scheduler <check-health/cli-checks-for-scheduler>`.
```

Comment:
```suggestion
  that it can be restarted even if it is in good condition. For greater confidence, consider using :ref:`CLI Check for Scheduler <check-health/cli-checks-for-scheduler>` or :ref:`Scheduler Health Check Server <check-health/scheduler-health-check-server>`.
```